components/fraudlabs_pro/actions/feedback-order/feedback-order.mjs

```diff
@@ -0,0 +1,52 @@
+import fraudlabsProApp from "../../fraudlabs_pro.app.mjs";
+
+export default {
+  name: "Feedback Order",
+  description: "Update status of a transaction from pending-manual-review to APPROVE, REJECT or IGNORE. The FraudLabs Pro algorithm will improve the formula in determine the FraudLabs Pro score using the data collected. Please refer to the [documentation](https://www.fraudlabspro.com/developer/api/feedback-order) for the explanation of the result returned.",
+  key: "fraudlabs_pro-feedback-order",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    fraudlabsProApp,
+    id: {
+      type: "string",
+      label: "ID",
+      description: "Unique transaction Id generated by Fraud Check API.",
+    },
+    format: {
+      propDefinition: [
+        fraudlabsProApp,
+        "format",
+      ],
+    },
+    action: {
+      type: "string",
+      label: "action",
+      description: "Perform APPROVE, REJECT, or REJECT_BLACKLIST action to transaction. Available values are: `APPORVE` which will approve a transaction, `REJECT` which will reject a transaction, and `REJECT_BLACKLIST` which will reject and blacklist the transaction. When a transaction was blacklisted, the information will be recorded into the system for future blacklist validation check.",
```

Comment:
I recommend using `options` here to make it easier for the user to select the right action. For example,
```
      options: [
        "APPROVE",
        "REJECT",
        "REJECT_BLACKLIST",
      ],
```
Then you can also shorten the description since you won't have to list all of the available values. Or you can even define labels and values here to make it more seamless ([see the docs](https://pipedream.com/docs/components/api/#component-api) for an example).